src/loaders/ObjectLoader.js

```diff
@@ -632,11 +679,73 @@ Object.assign( ObjectLoader.prototype, {
 			if ( data.visible !== undefined ) object.visible = data.visible;
 			if ( data.userData !== undefined ) object.userData = data.userData;
 
-			if ( data.children !== undefined ) {
+			/*
+			 * SkinnedMesh creates Bone instances as its children in the constructor
+			 * so skip Bone instance creation here.
+			 */
+			if ( data.type === 'SkinnedMesh' ) {
+
+				var scope = this;
+
+				function traverse ( data ) {
+
+					if ( data.children !== undefined ) {
 
-				for ( var child in data.children ) {
+						for ( var child in data.children ) {
 
-					object.add( this.parseObject( data.children[ child ], geometries, materials ) );
+							if ( data.children[ child ].type !== 'Bone' ) {
+
+								var parent;
+
+								if( data.type === 'SkinnedMesh' ) {
+
+									parent = object;
+
+								} else if ( data.type === 'Bone' ) {
+
+									// should make a method THREE.Skeleton.findBoneByName() ?
+									for ( var i = 0, il = object.skeleton.bones.length; i < il; i ++ ) {
+
+										if ( object.skeleton.bones[ i ].name === data.name ) {
+
+											parent = object.skeleton.bones[ i ];
+											break;
+
+										}
+
+									}
+
+								}
+
+								if ( parent ) {
+
+									parent.add( scope.parseObject( data.children[ child ], geometries, materials, skeletons ) );
+
+								}
+
+							} else {
+
+								traverse( data.children[ child ] );
+
+							}
+
+						}
+
+					}
+
+				}
+
+				traverse( data );
+
+			} else {
```

Comment:
Two things to keep in mind is that often you want characters to be able to pick something up in one of their hands (which is defined by bones.)  UE4 introduces the idea of sockets for that: https://docs.unrealengine.com/latest/INT/Engine/Content/Types/SkeletalMeshes/Sockets/  While I believe in Unity you can access the bone transform as this answer suggests: http://answers.unity3d.com/questions/182261/holding-an-object.html

The other thing people will want to do in the future is to manually place feet on the ground via IK, basically again modifying a bone.  I think in UE again this is special cased.  Not sure how it is achieved in Unity3D.

Just two things to keep in mind, but again they are future things...